tests/db_engine_specs/base_tests.py

```diff
@@ -17,7 +17,6 @@
 # isort:skip_file
 from datetime import datetime
 
-from tests.test_app import app
```

Comment:
This needs to be added back, as it's needed by tests.

Reply:
ok. added back as soon